### DISCUSSION TITLE
Textual SQL expression

### DISCUSSION THREAD

**Discussion by sindhubha** [fastapi/fastapi#9506] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
query = f'''select * from my table'''
```

### Description

Textual SQL expression 'SELECT * FROM my table...' should be explicitly declared as text('SELECT * FROM my table...')

### Operating System

Windows

### FastAPI Version

0.95.1

### Python Version

3.9.12

### Additional Context

in sqlalchemy  is this necessary to use text of

is there any other option without using text

**Comment by abhint** [comment, marked as answer]
```py
from sqlalchemy import text

query = text('SELECT * FROM `my table`')
```



If you are using sqlmodel please ask questions here: https://github.com/tiangolo/sqlmodel/discussions
sqlalchemy: https://github.com/sqlalchemy/sqlalchemy/discussions
